js-apps/meep-frontend/src/js/util/elem-utils.js

```diff
@@ -135,8 +135,18 @@ export const setElemFieldErr = (elem, field, err) => {
   }
 };
 
+export const resetElem = (elem) => {
```

Comment:
Does the resetElem only have use for resting the color editor part? 
If so, shouldn't we rename it for its only purpose.  (If it only restes states but only one state exists, should be called resetElemStates)
If not, should we had other data that needs to be reseted